fuel-crypto/src/ed25519.rs

```diff
@@ -0,0 +1,33 @@
+//! ED25519 signature verification
+
```

Comment:
should we rexport the dalek lib, (maybe even under test-helpers)? That way applications using ed25519 don't have to worry about importing the correct version of dalek or pontentially compiling it twice.